core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -6524,6 +6524,15 @@ public static List reverse(List list) {
     return list;
   }
 
+  /** Support the ARRAY_SLICE function. */
```

Comment:
Be consistent with the descriptions of other functions.
`/** SQL {@ xxxxxx }. */`

Reply:
Done